TheMovieDB/TheMovieDB/MVP/Views/MovieDetailVC.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MovieDetailVC.swift
+//  TheMovieDB
+//
+//  Created by Mariana Andrea Orman Berch on 22/9/21.
+//
+
+import Foundation
+import UIKit
+
+class MovieDetailVC: UIViewController {
+    
+    @IBOutlet weak var image: UIImageView!
+    
+    @IBOutlet weak var titleMovie: UILabel!
+    
+    @IBOutlet weak var releaseDate: UILabel!
+    
+    @IBOutlet weak var overview: UILabel!
+    
+    @IBOutlet weak var rating: UILabel!
+    
+    
+    var homeScreenVC = HomeScreenVC()
+    
+    func setValues () {
+
+        releaseDate.text = "Release Date: \(String(describing: homeScreenVC.selectedMovie?.releaseDate))"
```

Comment:
This will not work ever, you need to pass the data to this instance instead of trying to create another

Reply:
Haven't started on this yet